Sources/SwiftProtobuf/Google_Protobuf_Any.swift

```diff
@@ -41,6 +41,16 @@ fileprivate func typeName(fromMessage message: Message) -> String {
 }
 
 public extension Message {
+
+  /// Assumes that the data provided when creating the `Google_Protobuf_Any`
+  /// instance is encoded in the default manner, whether's in in binary
+  /// protobuf or JSON formats.
+  ///
+  /// Can be overridden to change how the data should be interpreted. See
```

Comment:
Actually, this should never be overridden.  It's purely a convenience wrapper for `Google_Protobuf_Any.unpackTo(target:)`.